spec/notifications/new_logbook_entry_created_notification_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe(NewLogbookEntryCreatedNotification, type: :model) do
+  context "when logbook entries are created" do
+    it "creates database notifications" do
+      # Two notifications expected because a User must exist for a Logbook_Entry to be created
+      expect { create(:logbook_entry) }.to(change(Notification, :count).by(2))
+    end
+
+    it "creates database notifications of the right type" do
+      create(:logbook_entry)
+      # the first notification will be for the user created, the second notification will pertain to the logbook
+      expect(Notification.all[1].type).to(eq("NewLogbookEntryCreatedNotification"))
+    end
```

Comment:
Not sure if it's worth checking the type for every single notification type.

Reply:
No,  I don't think so, usually in the first test we try the types of data.